README.rst

```diff
@@ -516,6 +516,51 @@ severity level.
          " than 1024, when you run the program with --v=1 or more. ";
          "Present occurence is " << google::COUNTER;
 
+
+Custom log prefix format
+~~~~~~~~~~~~~~~~~~~~~~~~
+glog supports changing the format of the prefix attached to log messages
+by receiving a user-provided callback to be used to generate such strings.
+That feature must be enabled at compile time by the ``WITH_CUSTOM_PREFIX``
+flag.
+
+For each log entry, the callback will be invoked with a ``LogMessageInfo``
+struct containing the severity, filename, line number, thread ID, and time of
+the event. It will also be given a reference to the output stream, whose
+contents will be prepended to the actual message in the final log line.
+
+For example:
+
+   .. code:: cpp
+
+      /* This function writes a prefix that matches glog's default format.
+       * (The third parameter can be used to receive user-supplied data, and is
+       * NULL by default.)
+       */
+      void PrefixAttacher(std::ostream &s, const LogMessageInfo &l, void*) {
```

Comment:
I would change the name to `Custom(Log)Prefix` or `MyCustom(Log)Prefix` with `Log` being optional. `PrefixAttacher` just sounds somewhat awkward to me.